apps/crn-frontend/src/discover/__tests__/Tutorials.test.tsx

```diff
@@ -27,7 +48,20 @@ const renderDiscoverTutorials = async (user: Partial<User>) => {
       >
         <Auth0Provider user={user}>
           <WhenReady>
-            <Tutorials />
+            <MemoryRouter initialEntries={[{ pathname: discover({}).$ }]}>
+              <Route path={discover.template}>
+                <Tutorials />
+              </Route>
+              <Route
+                path={
+                  discover.template +
+                  discover({}).tutorials.template +
+                  discover({}).tutorials({}).tutorial.template
+                }
+              >
+                <Tutorial />
```

Comment:
To cover the routes coverage complaint you will need to add a test which loads the Tutorials routing in the Dashboard.tsx.ts having had a quick look this could be apps/crn-frontend/src/discover/__tests__/Routes.test.tsx

Reply:
Thanks! I moved the test in this commit https://github.com/yldio/asap-hub/pull/2143/commits/681d5c4acf475c23e24e3a9d8f2d0506f4846bd7 :)